Client/mods/deathmatch/logic/luadefs/CLuaElementDefs.cpp

```diff
@@ -134,6 +135,7 @@ void CLuaElementDefs::AddClass(lua_State* luaVM)
     lua_classfunction(luaVM, "isStreamedIn", "isElementStreamedIn");
     lua_classfunction(luaVM, "isStreamable", "isElementStreamable");
     lua_classfunction(luaVM, "isLocal", "isElementLocal");
+    lua_classfunction(luaVM, "isData", "isElementData");
```

Comment:
The name doesnt fit, and is confusing: ```ped:isData("data")``` seems pretty confusing to me, **hasData** would be better imho.

Reply:
I agree